src/Tribe/Integrations/WPML/Meta.php

```diff
@@ -33,6 +34,8 @@ public function translate_post_id( $value, $object_id, $meta_key, $single ) {
 			return $value;
 		}
 
+		$object_id = Occurrence::normalize_id( $object_id );
```

Comment:
This class will be defined, but will work incorrectly when not in Custom Tables v1 context.
I would wrap this in a check for CT1 full activation:
```suggestion
         if( tribe()->getVar( 'ct1_fully_activated' ) ) {
		$object_id = Occurrence::normalize_id( $object_id );
	}
```

Reply:
Thanks, added.